doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
 ---
-title: $PlatformShort$ Charts | Data Visualization | Infragistics
-_description: Infragistics' $PlatformShort$ Charts
+title: $PlatformShort$ Charts & Graphs Library | Data Visualization | Infragistics | Ignite UI for $PlatformShort$
+_description: Visualize any type of data, business, financial, scientific, and scatter, with the capability of millisecond updates with millions of data points in your $PlatformShort$ apps.
 _keywords: $PlatformShort$ Charts, Chart, Infragistics
 ---
-# $PlatformShort$ Charts
-
-<!-- TODO add introduction to charts -->
-
-## Types of $PlatformShort$ Charts
-
-- [Area Chart](chart-types-area.md)
-- [Bar Chart](chart-types-bar.md)
-- [Bubble Chart](chart-types-scatter.md#$PlatformShort$-Scatter-Bubble-Chart)
-- [Column Chart](chart-types-column.md)
-- [Donut Chart](../doughnut-chart.md)
-- [Financial Chart](chart-types-financial.md)
-- [Line Chart](chart-types-line.md)
-- [Polar Chart](chart-types-polar.md)
-- [Pie Chart](../pie-chart.md)
-- [Radial Chart](chart-types-radial.md)
-- [Scatter Chart](chart-types-scatter.md)
-- [Shape Chart](chart-types-scatter.md#$PlatformShort$-Scatter-Polygon-Chart)
-- [Stacked Chart](chart-types-area.md#$PlatformShort$-Stacked-Area-Chart)
-- [Spline Chart](chart-types-spline.md)
-- [Step Chart](chart-types-step.md)
-- [Sparkline](../sparkline.md)
-- [Treemap](../treemap-overview.md)
-
-## List of Chart Features
-
-<!-- tooltips, legends, axes, overlays, annotations, etc. -->
\ No newline at end of file
+# $PlatformShort$ Charts & Graphs Overview
+
+Ignite UI for $PlatformShort$ Charts & Graphs is an extensive library of data visualizations that enable stunning, interactive charts and dashboards for your web and mobile apps. Built for speed and beauty, designed to work on every modern browser, and with complete touch and interactivity, you can quickly and easily build responsive visuals into your next app on any device. 
+
```

Comment:
I think we have to add some standout sample here so that we have something grasping page visitor attention above the fold.

Reply:
I agree with you @StefanIvanov . It incorporates different charts, so I think it will be a nice example. But I see that it's a Blazor sample, so I am not sure if it can be used for the other frameworks.